Makefile

```diff
@@ -193,6 +197,12 @@ ENVTEST ?= $(LOCALBIN)/setup-envtest
 KUSTOMIZE_VERSION ?= v5.2.1
 CONTROLLER_TOOLS_VERSION ?= v0.15.0
 
+.PHONY: kubectl
+kubectl: $(KUBECTL) ## Use envtest to download kubectl
+$(KUBECTL): $(LOCALBIN) $(ENVTEST)
+	KUBEBUILDER_ASSETS=$$($(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path); \
```

Comment:
This can be on one cmd, where KUBEBUILDER_ASSETS is an env var just for the command being executed, rather than two separate commands. But also, we could add a check here if KUBECTL exists, to only do this one. Although it can be enhanced to allow for updating the version. Either way, should also probably be a symlink rather than a cp

Reply:
The KUBEBUILDER_ASSETS environment variable is scoped to the single shell session where it is set, so there is no risk of leakage between commands. A check to see if KUBECTL exists is added and cp has been changed to a symlink.